src/content/docs/apm/agents/go-agent/configuration/go-agent-code-level-metrics-config.mdx

```diff
@@ -0,0 +1,286 @@
+---
+title: Go agent code-level metrics configuration
+tags:
+  - Agents
+  - Go agent
+  - Configuration
+metaDescription: Add source code context to your application traces, associating the trace data with the specific code being instrumented.
+---
+
+You can edit configuration settings for the Go agent to control the collection of code-level metrics. For general information about code-level metrics, see [Performance monitoring with CodeStream](/docs/codestream/how-use-codestream/performance-monitoring#code-level).
```

Comment:
This is super nitpicky--while the first sentence is to the point, clear, and direct, I think it lacks a call to action or motivation to continue reading. As someone who is new to this, I'd like something more like a statement of purpose.

Reply:
@homelessbirds, I revamped the introduction by moving the contents of the Overview section into the introduction.  I also broke up that long sentence to add clarity.